Assets/Scripts/LitMotion/PropsAnimation.cs

```diff
@@ -0,0 +1,100 @@
+using UnityEngine;
+using LitMotion;
+using LitMotion.Extensions;
+using Random = UnityEngine.Random;
+
+public class PropsAnimation : MonoBehaviour
+{
+    [Header("Movement Parameters (移動のパラメータ)")]
+    [Tooltip("XYZ軸方向へのランダムな移動の振幅。大きくすると大きく動きます。")]
+    public Vector3 movementAmplitude = new Vector3(0.5f, 0.5f, 0.5f);
+    
+    [Tooltip("アニメーションの基本的な持続時間 (秒)")]
+    public float baseDuration = 5f;
+    
+    [Tooltip("次のモーションに移行するまでのランダムな遅延の最大値 (秒)")]
+    public float maxDelay = 1f;
+
+    [Header("Rotation Parameters (回転のパラメータ)")]
+    [Tooltip("XYZ軸回転の速さ（度/秒）")]
+    public float rotationSpeed = 30f;
+
+    // 個々のオブジェクトのランダムな動きを生成するためのシード
+    private float randomSeed;
```

Comment:
シードとあるけど乱数そのものでは？
乱数のシードは乱数を作るための引数みたいなやつなので、ちょっと違うかなと
```suggestion
    private float randomValue;
```